app/controllers/admin/merchants_controller.rb

```diff
@@ -1,5 +1,5 @@
-class Admin::MerchantsController < ApplicationController
```

Comment:
Was this still operating fine on our tests without Admin::BaseController? Glad you caught it